src/neuroconv/datainterfaces/ecephys/baserecordingextractorinterface.py

```diff
@@ -14,7 +14,7 @@
 from ...utils import get_schema_from_hdmf_class, get_base_schema, OptionalFilePathType
 
 
-class BaseRecordingExtractorInterface(BaseDataInterface, ABC):
+class BaseRecordingExtractorInterface(BaseDataInterface):
```

Comment:
Requires removal of this mix-in to avoid metaclass conflicts.

Also, why is this mix-in defined anyhow on `main` since the parent is an `ABC` anyway?